doc/command_line_tool.md

```diff
@@ -38,15 +38,27 @@ In order to build the grpc command line tool from a fresh clone of the grpc
 repository, you need to run the following command to update submodules:
 
 ```
-git submodule update --init
+$ git submodule update --init
 ```
-
+---
+### Unix user
 You also need to have the gflags library installed on your system. On Linux
 systems, gflags can be installed with the following command:
 
 ```
-sudo apt-get install libgflags-dev
+$ sudo apt-get install libgflags-dev
+```
+
+### OSX user
+This is a C++ CLI, and you need gflags to build it.
```

Comment:
Replace with something like "On Mac, gflags can be installed via Homebrew:"

Reply:
done